Automattic/jetpack
# Social Previews v2 | Base branch/PR

All the PRs should be created against this branch - `feature/social-previews-v2`


Related https://github.com/Automattic/wp-calypso/pull/76808

## Proposed changes:
For everything expected here, please see the linked PRs

### Other information:


## Jetpack product discussion


pdrWKz-Ka-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:

* Checkout this PR
* In Jetpack, run `jetpack build plugins/jetpack`
* Start writing a new post with JT site via Jetpack
* Click Jetpack icon on the top of the sidebar
* Under "Social Previews", click on "Preview"
* Confirm that the previews work as expected



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1204571006046542
